code/game/machinery/vending.dm

```diff
@@ -804,7 +804,7 @@
 
 	prices = list(/obj/item/weapon/reagent_containers/food/drinks/smallmilk = 3,
 					/obj/item/weapon/reagent_containers/food/drinks/smallchocmilk = 3,
-					/obj/item/weapon/reagent_containers/food/drinks/glass2/fitnessflask/proteinshake = 20,
+					/obj/item/weapon/reagent_containers/food/drinks/glass2/fitnessflask/proteinshake = 40,
```

Comment:
If the effective food of the item is already being cut in half, why is the price being doubled? This seems excessive, double-nerfing it like this

Reply:
Probably to push players towards LiquidRations and leave protein to the people who need it, though LiquidRations are now just OP

Reply:
Agreed, and I added an adjustment to the liquid rations to where they're now mostly protein based, with only minor nutriments while we go through the changes.